app/views/items/index.html.erb

```diff
@@ -1,25 +1,27 @@
 <% if @merchant %>
-  <h1><%= link_to @merchant.name, "/merchants/#{@merchant.id}"%><span> Items</span></h1>
-  <p align="center"><%= link_to "Add New Item", "/merchants/#{@merchant.id}/items/new" %></p>
+  <h1><%= link_to @merchant.name, merchant_path(@merchant.id) %><span> Items</span></h1>
+  <p align="center"><%= link_to "Add New Item", new_merchant_item_path(@merchant.id) %></p>
 <% else %>
   <h1>All Items</h1>
 <% end %>
 <section class="grid-container">
   <% @items.each do |item| %>
-    <section class = "grid-item" id= 'item-<%=item.id%>'>
-      <h2> <%=link_to item.name, "/items/#{item.id}" %> </h2>
-      <p>Sold by: <%=link_to item.merchant.name, "/merchants/#{item.merchant.id}" %></p>
-      <img src= <%= item.image %>>
-      <p> <%= item.description unless @merchant%> </p>
-      <p>Price: <%=number_to_currency(item.price) %> </p>
-      <p>Inventory: <%= item.inventory %> </p>
-      <% if !@merchant %>
+    <% if item.active? %>
+      <section class = "grid-item" id= 'item-<%=item.id%>'>
+        <h2> <%=link_to item.name, item_path(item.id) %> </h2>
+        <p>Sold by: <%=link_to item.merchant.name, merchant_path(item.merchant.id) %></p>
+        <%= link_to(image_tag(item.image), item_path(item.id)) %>
+        <p> <%= item.description unless @merchant%> </p>
+        <p>Price: <%=number_to_currency(item.price) %> </p>
+        <p>Inventory: <%= item.inventory %> </p>
+        <% if !@merchant %>
+        <% end %>
```

Comment:
Not sure what this does here. Is this a placeholder?

Reply:
Jon P asked about this yesterday in the channel and Meg replied that it could be deleted.